jumpscale/sals/nginx/templates/location_proxy.conf

```diff
@@ -11,4 +11,4 @@
     proxy_set_header X-Real-IP $remote_addr;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
-    proxy_pass {{ location.scheme }}://{{ location.host }}:{{ location.port }}{{ location.path_dest }}{{ "/" if not location.path_dest.endswith("/") }};
```

Comment:
make sure in threebot that we do `rstrip("/")` for `path_dest` and add `/` here at the end